src/main/java/com/synopsys/integration/alert/channel/jira/cloud/web/JiraCustomEndpoint.java

```diff
@@ -95,8 +96,6 @@ public JiraCustomEndpoint(JiraChannelKey jiraChannelKey, CustomEndpointManager c
             Thread.currentThread().interrupt();
             return Optional.of(responseFactory.createInternalServerErrorResponse("", "Thread was interrupted while validating Jira plugin installation: " + e.getMessage()));
         }
-
-        return Optional.empty();
```

Comment:
Having the `return Optional.empty()` after the try-catch makes it easy to forget to return/throw errors.